chain/chain/src/doomslug.rs

```diff
@@ -542,6 +613,13 @@ impl Doomslug {
         let _ = self.on_approval_message_internal(now, approval, stakes);
     }
 
+    /// Gets the current status of approvals for a given height.
+    /// It will only work for heights that we have in memory, that is that are not older than MAX_HEIGHTS_BEFORE_TO_STORE_APPROVALS
+    /// blocks from the head.
+    pub fn approval_status_at_height(&self, height:&BlockHeight) -> Option<ApprovalAtHeightStatus> {
```

Comment:
Does this have to return an Option? It seems that `ApprovalAtHeightStatus` has a reasonable neutral element of  no approvals?

Reply:
done.